examples/dax/cluster.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: dax.aws.crossplane.io/v1alpha1
+kind: Cluster
+metadata:
+  name: example-dax-cluster
+spec:
+  forProvider:
+    description: "This is an example dax cluster."
+    tags:
+      - key: "testKey"
+        value: "testValue"
+    sseSpecification:
+      enabled: true
+    securityGroupIDs:
+      - "sg-0ab0000ff000000d0"
+    iamRoleARN: "arn:aws:iam::999999999999:role/dax-us-east-1"
+    nodeType: "dax.t2.small"
+    region: "us-east-1"
+    replicationFactor: 2
+    parameterGroupName: "example-parameter-group"
+    subnetGroupName: "example-subnet-group"
+    availabilityZones:
+      - "us-east-1a"
+      - "us-east-1b"
+    preferredMaintenanceWindow: "tue:06:30-tue:07:30"
+    clusterEndpointEncryptionType: "NONE"
+---
+apiVersion: dax.aws.crossplane.io/v1alpha1
+kind: Cluster
+metadata:
+  name: example-dax-cluster-with-ref
+spec:
+  forProvider:
+    description: "This is an example dax cluster with references."
+    tags:
+      - key: "testKeyRef"
+        value: "testValueRef"
+    sseSpecification:
+      enabled: false
+    securityGroupIDs:
+      - "sg-0ab0000ff000000d0"
+    iamRoleARN: "arn:aws:iam::999999999999:role/dax-us-east-1"
+    nodeType: "dax.t2.small"
+    region: "us-east-1"
+    replicationFactor: 1
+    subnetGroupNameRef:
+      name: "example-subnet-group"
+    parameterGroupNameRef:
+      name: "example-parameter-group"
+    availabilityZones:
+      - "us-east-1a"
+    preferredMaintenanceWindow: "tue:06:30-tue:07:30"
+    clusterEndpointEncryptionType: "NONE"
```

Comment:
```suggestion
    clusterEndpointEncryptionType: "NONE"
 
```

Reply:
Thanks, fixed in [`0aac4`](https://github.com/crossplane/provider-aws/pull/1256/commits/0aac42d58a30c4e20b55d6810dc00363ba1ab2c2).